pkg/metrics/otel.go

```diff
@@ -165,10 +165,15 @@ processors:
       # These labels are useful to users, but too noisy for global aggregation.
       - key: commit
         action: delete
-      - key: type
-        action: delete
   metricstransform/kubernetes:
     transforms:
+      - include: api_duration_seconds
+        action: update
+        operations:
+          # Eliminate 'type' tag from metric using aggregation
```

Comment:
nit: maybe just "Remove" instead of "Eliminate"

Reply:
maybe also note that it's high cardinality as the reason

Reply:
Done!